src/react-components/src/components/03-organisms/SearchPromo/SearchPromo.tsx

```diff
@@ -0,0 +1,51 @@
+//OF-21 Search Promo
+
+import * as React from "react";
+import bem from "../../../utils/bem";
+import SectionTitle from "../../02-molecules/Headings/SectionTitle";
+import SearchBar from "../../02-molecules/SearchBar/SearchBar";
+import BodyText from "../../01-atoms/Text/Text/BodyText";
+
+export interface SearchPromoProps {
+  headingText: string;
+  modifiers?: [];
+  blockName?: string;
+  titleId: string;
+  searchBarId: string;
+  searchButtonId: string;
+  dropdownId: string;
+  hasError?: boolean;
+  errorMessage?: string;
+  searchDropdownOptions: string[];
+  advancedSearchMessage: JSX.Element;
+  selectChangeHandler: (event: React.FormEvent) => void;
+  selectBlurHandler: (event: React.FormEvent) => void;
+  searchSubmitHandler: (event: React.MouseEvent) => void;
+  textChangeHandler?: (event: React.FormEvent) => void;
+}
+export default function SearchPromo(props: SearchPromoProps) {
+  const { headingText, modifiers, blockName, titleId, searchBarId, searchButtonId, hasError, errorMessage,
+    dropdownId, searchDropdownOptions, advancedSearchMessage, 
+    selectChangeHandler, selectBlurHandler, searchSubmitHandler, textChangeHandler } = props;
+
+  const searchpromo__base_class = "search-promo";
+
+
+  return <div className={bem(searchpromo__base_class, [], blockName)}>
+    <div className={bem('content', [], searchpromo__base_class)}>
```

Comment:
Is there a reason why there must be two divs here?

Reply:
The outer div handles the the `breakout` that extends the `background-color` while the inner div handles keeping the content on the grid.